src/routes/transactions/mappers/module-transactions/module-transaction-details.mapper.spec.ts

```diff
@@ -109,7 +109,7 @@ describe('ModuleTransactionDetails mapper (Unit)', () => {
       trustedDelegateCallTarget,
     );
 
-    const actual = await mapper.mapDetails(chainId, transaction);
+    const actual = await mapper.mapDetails(chainId, transaction, 0);
```

Comment:
Is there any reason to pass `0` in these tests? I mean, could it be an opportunity to test that `timezoneOffsetMs` is actually added to the value in the response's `executedAt` property?

Reply:
I agree. Although `0` was used else the expectation of the test would have to be changed. I've added new tests instead. [8fad952](https://github.com/safe-global/safe-client-gateway/pull/927/commits/8fad9525e2dfa87f90facd261053503823e2983b)